components/incoming/Topcomponent.vue

```diff
@@ -1,7 +1,7 @@
 <template>
  <div class="icx-top-container">
    <div class="icx-top">
-      <img src="../assets/product_GT-horizontal-white.png" alt="GT-logo">
+      <img src="@/assets/product_GT-horizontal-white.png" alt="GT-logo">
```

Comment:
```suggestion
      <img src="@/assets/images/product_GT-horizontal-white.png" alt="GT-logo">
```
template部はassets/imagesに変更お願いします。